ui-kit-lib/src/main/kotlin/com/linkedplanet/uikit/wrapper/atlaskit/datetimepicker/DateTimeRangeComponent.kt

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2022 linked-planet GmbH.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedplanet.uikit.wrapper.atlaskit.datetimepicker
+
+import com.linkedplanet.uikit.wrapper.atlaskit.calendar.Calendar
+import moment.Moment
+import moment.moment
+import react.*
+
+
+external interface DateTimeRangeProps: Props {
+    var minDate: String?
+    var maxDate: String?
+    var disabledDates: Array<String>
+    var startDate: String?
+    var endDate: String?
+    var locale: String  // "de-de"
+    var weekStartDay: Int // 1
+    var onChange: (String, String) -> Unit
+    var onCollision: () -> Unit
+}
+
+val DateTimeRange = fc<DateTimeRangeProps> { props ->
+
+    val (dateSelectCounter, setDateSelectCounter) = useState(0)
```

Comment:
hier sollten wir uns mal einigen ob wir stattdessen lieber 
```suggestion
    var dateSelectCounter by useState(0)
```
schreiben wollen. Das ist das Kotlin idiom und erlaubt es danach die variablen, wie normale kotlin vars zu verwenden

Reply:
Done